### PR TITLE
chore: new Windows signed scripts and package version

### PR DESCRIPTION
**Pull request by haofan-ms**
**Reason for Change**:

Updated Windows provisioning scripts to include latest changes, and update zip package version and source to use ASH team's storage account.

AgentBaker repo reference commit:
https://github.com/Azure/AgentBaker/commit/dfa43b82c6393867d60cc24412046bdb10e08506
v0.0.29

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/azenvtypes.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ var (
 		VnetCNIWindowsPluginsDownloadURL:     "https://kubernetesartifacts.azureedge.net/azure-cni/" + AzureCniPluginVerWindows + "/binaries/azure-vnet-cni-singletenancy-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
 		ContainerdDownloadURLBase:            "https://storage.googleapis.com/cri-containerd-release/",
 		CSIProxyDownloadURL:                  "https://kubernetesartifacts.azureedge.net/csi-proxy/" + DefaultWindowsCsiProxyVersion + "/binaries/csi-proxy-" + DefaultWindowsCsiProxyVersion + ".tar.gz",
-		WindowsProvisioningScriptsPackageURL: "https://kubernetesartifacts.azureedge.net/aks-engine/windows/provisioning/signedscripts-" + DefaultWindowsProvisioningScriptsPackageVersion + ".zip",
+		WindowsProvisioningScriptsPackageURL: "https://akseashpackages.blob.core.windows.net/scripts/signedscripts-" + DefaultWindowsProvisioningScriptsPackageVersion + ".zip",
```
Comment (jadarsie) [inline pkg/api/azenvtypes.go:83]: maybe a pain to change at this point, but I like `akseashartifacts` better

Comment (jadarsie) [reply pkg/api/azenvtypes.go:83]: on the same note, the path `/windows/provisioning` also seems more appropriated.

File: scripts/build-windows-provisioning-scripts.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@ required_env_vars=(
 
 unset_vars=()
 for v in "${required_env_vars[@]}"
-do
+do
```
Comment (jadarsie) [inline scripts/build-windows-provisioning-scripts.sh:22]: extra white space

File: pkg/api/azenvtypes.go (hunk 1 above)
Comment (haofan-ms) [reply pkg/api/azenvtypes.go]: Update

**Review by christinalau0 — DISMISSED** [review]

File: parts/k8s/windowskubeletfunc.ps1 [hunk 3]
```diff
@@ -203,47 +169,11 @@ New-InfraContainer {
...
+            Invoke-Executable -Executable "ctr" -ArgList @("-n", "k8s.io", "image", "pull", "$defaultPauseImage") -Retries 5 -RetryDelaySeconds 30
         }
-    }
-    else {
-        Build-PauseContainer -WindowsBase "mcr.microsoft.com/nanoserver-insider" -DestinationTag $DestinationTag -ContainerRuntime $ContainerRuntime
-    }
-}
-
-function
-Test-ContainerImageExists {
```
Comment (jadarsie) [inline parts/k8s/windowskubeletfunc.ps1:226]: Temporary marker, `Test-ContainerImageExists` should be kept

**Review by christinalau0 — APPROVED** [review]

**Pull request merged**